_data/vendors.yml

```diff
@@ -131,6 +131,14 @@
   percent_increase: 375%
   pricing_source: https://about.gitlab.com/pricing
   updated_at: 2018-10-21
+  
+- name: Hashicorp (Terraform Cloud)
+  url: https://www.terraform.io/
+  base_pricing: $20 per u/m
+  sso_pricing: Call Us!
+  percent_increase: ??
```

Comment:
```suggestion
  sso_pricing: $70 per u/m
  percent_increase: 350%
```

If you sign up for a free trial and go to upgrade you get this page:

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/597907/115119048-76051780-9f74-11eb-9a2a-b4e21c07fd86.png">

I guess the main pricing page says "Call us!" because the truth is embarrassing.